recipe/meta.yaml

```diff
@@ -16,6 +16,7 @@ build:
 requirements:
   build:
     - python
+    - pip
```

Comment:
Yeah, I'm a bit surprised it made a difference. It's not in other recipes with the same install command...